src/Skin.js

```diff
@@ -171,6 +171,42 @@ class Skin extends EventEmitter {
      */
     updateSilhouette () {}
 
+    /**
+     * Set the contents of this skin to an empty skin.
+     * @fires Skin.event:WasAltered
+     */
+    setEmptyImageData () {
+        // Free up the current reference to the _texture
+        this._texture = null;
+
+        if (this._emptyImageData) {
+            this._silhouette.update(this._emptyImageData);
+        } else {
+            // Create a transparent pixel
+            this._emptyImageData = new ImageData(1, 1);
+
+            // Create a new texture and update the silhouette
+            const gl = this._renderer.gl;
+
+            const textureOptions = {
+                auto: true,
+                wrap: gl.CLAMP_TO_EDGE,
+                src: this._emptyImageData
+            };
+
+            // Note: we're using _emptyImageTexture here instead of _texture
+            // so that we can cache this empty texture for later use as needed.
+            // this._texture can get modified by other skins (e.g. BitmapSkin
+            // and SVGSkin, so we can't use that same field for caching)
+            this._emptyImageTexture = twgl.createTexture(gl, textureOptions);
+            this._silhouette.update(this._emptyImageData);
+        }
+
+        // TODO should we just make it not visible here?
```

Comment:
I think so, personally. It will make the renderer skip a draw it doesn't have to do

Reply:
Hmmm... just realized my todo comment might be misleading. I think there might not be a concept of a skin being visible or not. Just the drawable.

Reply:
That's correct - there's no concept of a skin being invisible. In general I think this situation is likely to be temporary so I think it's OK to "waste" the draw call(s) that correspond to an empty skin. ~If~When someone makes a project which intentionally uses this path in the long term there'll be a small performance penalty but really the project should run slightly faster than it would with just about any other skin, so I think that's OK too.